lab_3_keywords_textrank/main.py

```diff
@@ -120,7 +130,11 @@ def _learn_indices(self, tokens: tuple[str, ...]) -> None:
             tokens : tuple[str, ...]
                 sequence of string tokens
         """
-        pass
+        self._word2id = {token: randint(1000, 9999) for token in tokens}
```

Comment:
for different tokens you can get equals ints

Reply:
Done